monitor/grafana/zeebe.json

```diff
@@ -23925,4 +23925,4 @@
   "uid": "zeebe-dashboard",
   "version": 34,
   "weekStart": ""
-}
\ No newline at end of file
+}
```

Comment:
We keep having these diffs with the newline at the end of the file. I don't understand why is that

Reply:
The previous version was not ending with a new line, the new version is (nothing is removed here)

Usually, text editors keep (or most likely, add here) the ending new line ; it can get removed when piping the content into a CLI tool (or exporting it from a tool for instance)

Good to have this ending new line though, generally speaking :)

Reply:
My suspicion is that grizzly is removing that newline when editing locally :thought_balloon: